gulpfile.js

```diff
@@ -175,7 +170,7 @@ const serve = cb => {
   });
 };
 
-const build = gulp.series(plPhp, copyUswdsFonts, copyUswdsImages, copyIconSprite, plCss, plJs);
+const build = gulp.series(copyUswdsFonts, copyUswdsImages, copyIconSprite, plCss, plJs);
```

Comment:
@sleepingmonk This doesn't break the watch functionality?

Reply:
Hm.  Didn't test/consider that.  My objective was just to get a usable artifact out of the working repo that a theme can consume as a package.  I wasn't concerned about live dev.  :\

My artifact builder has a patch tool built in to deal with this so can probably just continue using that.  I might play around with it some more.

Thanks for the review @jacine!